WebView/app/src/main/assets/main.js

```diff
@@ -25,7 +25,11 @@ function getData() {
 		var form = document.getElementById("location");
 		var currentLocation = form.options[form.selectedIndex].value;
 		document.getElementById("title").innerText = form.options[form.selectedIndex].text;
-        document.getElementById("description").innerText = data[currentLocation].description;
+        document.getElementById("currentTemp").innerText = data[currentLocation].currentTemp;
```

Comment:
Did we fix the indentation for this in #147?

Reply:
It does not look like this was changed in that commit however it is at the correct indentation in my local version. I am unsure why it would shift back here?

Reply:
It sounds like your editor might be converting whitespace characters, or does not clearly distinguish whitespace characters for you. If you have a unix machine, `od -c path/to/main.js` would help distinguish which whitespace characters are actually in use. You may be able to reconfigure your editor to help fix whitespace, but I'm not familiar with how to do this in Android Studio.

It looks like the `webview` branch is currently inconsistent with tabs vs. spaces, so let's not worry about this for this PR. But please send a follow-up to switch the full file to spaces per the google styleguide.